client/Pages/EventTypeCreate/View.elm

```diff
@@ -330,7 +330,7 @@ viewForm model setup =
                     Help.options
                     Optional
                     Enabled
-                    [ "1", "2", "3", "4" ]
+                    (String.split " " model.userStore.user.settings.retentionTimeDaysValues)
```

Comment:
It is better to move it to line 180 and use just the resulting variable here.
Plus I would use pipe here 
```
retentionTimeDaysValues =  model.userStore.user.settings.retentionTimeDaysValues |> String.split " "
```

Reply:
@SergKam https://github.com/zalando-nakadi/nakadi-ui/pull/101/commits/641ce62d79454177975b87f6cdd0b34e4a67c8c4